src/frontend/styles/components/sidebar.pcss

```diff
@@ -1,6 +1,34 @@
 .docs-sidebar {
   width: 100vw;
-    
+
+  &--animated {
+    .docs-sidebar__content {
+      transition: transform 200ms ease-in-out;
+    }
+
+    .docs-sidebar__slider {
+      transition: left 200ms ease-in-out;
+
+      svg {
```

Comment:
lets remove transition from the SVG sice the whole button is moving at this moment, and the svg rotation animation is looks not so well.

Reply:
Fixed